schema.sql

```diff
@@ -86,4 +86,18 @@ ALTER TABLE visits ADD COLUMN visit_date date;
 
 /*Drop primary key constraint in visits table*/
 
-ALTER TABLE visits DROP CONSTRAINT visits_pkey;
\ No newline at end of file
+ALTER TABLE visits DROP CONSTRAINT visits_pkey;
+
+-- Add an email column to your owners table
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+
+--  Improve execution time of queries with index creation.
+CREATE INDEX animal_id_indx ON animals(animal_id ASC);
+
+CREATE INDEX vet_id_indx ON visits(vet_id ASC);
+
+CREATE INDEX email_indx ON owners(email ASC);
```

Comment:
[OPTIONAL]
- [ ] This is a nice-to-have suggestion as it won't improve the query that was reported as problematic in the requirements for this project.
  As email addresses are not case-sensitive, we might need to look for the emails in a case-insensitive way. There is an option to reflect that in the index by using:

```sql
CREATE INDEX ON owners ((lower(email)));
```